data/secscan_model/secscan_v4_model.py

```
@@ -110,6 +110,32 @@ def garbage_collect_manifest_report(self, manifest_digest):
 
 
+def _has_container_layers(layers):
+    """
+    Checks if the layers are proper container image layers. For images that do not contain image layers, such as
+    Singularity containers, Helm charts, artifacts and similar, we skip the scanning process.
+    """
+    from image.docker.schema2 import DOCKER_SCHEMA2_LAYER_CONTENT_TYPE
+    from image.docker.schema2.manifest import DockerV2ManifestImageLayer
+    from image.oci import OCI_IMAGE_LAYER_CONTENT_TYPES
+
+    CONTAINER_LAYER_TYPES = set(OCI_IMAGE_LAYER_CONTENT_TYPES + [DOCKER_SCHEMA2_LAYER_CONTENT_TYPE])
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)